tables/sw-ke-g1-2.ctb

```diff
@@ -4,21 +4,26 @@
 #-display-name: Swahili (Kenya) Grade 1.2
 #
 #-author-name: Jake Kyle
+#-author-name: Dipendra Manocha
+#-author-name: Martin Kieti
 #-maintainer-name: Jake Kyle
 #-maintainer-email: jake@compassbraille.org
 #-updated: 2022
 #
 #+language:sw
-#+region:ke
+#+region:sw-KE
 #+type:literary
-#+grade:1.2
+#+grade:1
```

Comment:
This change was probably unintended.

Reply:
Yes - just noticed that! - fixing now whilst adding contraction metadata